README.rst

```diff
@@ -1,11 +1,72 @@
-This package provides an ``ExplainCollection`` class
-that allows that allows PyMongo's Collection methods to be explained_
+==============
+PyMongoExplain
+==============
+
+:Info: Explain collections in PyMongo. See
+       `GitHub <https://github.com/mongodb-labs/pymongoexplain>`_
+       for the latest source.
+:Author: Julius Park
+
+About
+=====
+This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
 .. _explained: https://docs.mongodb.com/master/reference/command/explain/#dbcmd.explain.
 
 
+Support / Feedback
+==================
+
+For issues with, questions about, or feedback for PyMongoExplain, please look into
+our `support channels <http://www.mongodb.org/about/support>`_. Please
+do not email any of the PyMongoExplain developers directly with issues or
+questions - you're more likely to get an answer on the `mongodb-user
+<http://groups.google.com/group/mongodb-user>`_ list on Google Groups.
+
+Bugs / Feature Requests
+=======================
+
+Think you’ve found a bug? Want to see a new feature in PyMongoExplain?
+Please open a case in our issue management tool, JIRA:
+
+- `Create an account and login <https://jira.mongodb.org>`_.
```

Comment:
We might want to consider letting people open issues against pymongoexplain in GitHub itself? We do need to figure out where we want to track pymongoexplain-related issues in the long-term. CC: @ShaneHarvey

Reply:
Let's just use github issues here. We're already using them to track issues and improvements.

Reply:
Agreed. Lets stick with GitHub for now. Julius - please update this text to ask users to file issue tickets under the GitHub project itself. Please remove all references to JIRA.

Reply:
Done.